frontend/__tests__/utils/functions.js

```diff
@@ -53,12 +53,12 @@ test('Validate Files should only return allowed file types', () => {
   }]);
 });
 
-test('Get File size should convert bytes to MB', () => {
+test('Get File size should convert bytes to mb', () => {
```

Comment:
the pedant in me associates a lower case b with bits, not bytes. Shouldn't it be kB?